recipe/meta.yaml

```diff
@@ -42,6 +51,9 @@ requirements:
      - typing_extensions {{ typing_extensions }}        #[py<38]
      - importlib_resources {{ importlib_resources }}    #[py<39]
      - tensorflow-base {{ tensorflow }}
+     # Use pins to control cos6/cos7 match
```

Comment:
This is the same as TensorFlow hub where a compiler is needed at build time, but no compiled code ends up in the package. So we don't want to add a run dep here on the libgcc libraries. And perhaps this PR is unnecessary.